frontend/src/components/BackpackUnit.tsx

```diff
@@ -2,20 +2,39 @@ import React from "react";
 import { useDrag } from "react-dnd";
 import { Unit as UnitType } from "../models/game-state.model";
 import Unit from "./Unit";
+import { Button } from "reactstrap";
+import { UNIT_SELL_PRICE_MULTIPLIER } from "../constants";
 
 interface Props {
   unit: UnitType;
+  sellUnit: () => void;
 }
 
-const BackpackUnit: React.FunctionComponent<Props> = ({ unit }: Props) => {
+const BackpackUnit: React.FunctionComponent<Props> = ({ unit, sellUnit }: Props) => {
   const drag = useDrag({
     item: {
       type: "backpackUnit",
       id: unit.id,
     },
   })[1];
 
-  return <Unit unit={unit} dragRef={drag} />;
+  const sellingPrice = unit.price * UNIT_SELL_PRICE_MULTIPLIER;
```

Comment:
probably a good idea to add some sort of rounding here to fight floating point arithmetics.